Makefile

```diff
@@ -142,6 +150,83 @@ calico/felix: bin/calico-felix
 	cp bin/calico-felix docker-image/bin/
 	docker build -t calico/felix docker-image
 
+# Targets for Felix testing with the k8s backend and a k8s API server,
+# with k8s model resources being injected by a separate test client.
+LOCAL_IP_ENV?=$(shell ip route get 8.8.8.8 | head -1 | awk '{print $$7}')
+K8S_VERSION=1.5.3
+FELIX_K8S=felix-k8s
+.PHONY: k8s-fv-test run-k8s-apiserver stop-k8s-apiserver run-etcd stop-etcd
+k8s-fv-test: calico/felix run-k8s-apiserver k8sfv/k8sfv.test
+	@-docker rm -f $(FELIX_K8S)
+	sleep 1
+	docker run --detach --privileged --name=$(FELIX_K8S) \
+	-e FELIX_LOGSEVERITYSCREEN=info \
+	-e FELIX_DATASTORETYPE=kubernetes \
+	-e FELIX_PROMETHEUSMETRICSENABLED=true \
+	-e K8S_API_ENDPOINT=https://$(LOCAL_IP_ENV):6443 \
+	-e K8S_INSECURE_SKIP_TLS_VERIFY=true \
```

Comment:
Good chance that TLS slows us down, maybe we need to turn it on?  It's the norm in k8s-land.